beps/bep_0003.rst

```diff
@@ -87,70 +92,153 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layers``
```

Comment:
What is the underlying motivation for that? To be able to transform a hybrid torrent to the new format exclusively without changing the new hash? I guess that's possible, but it would prevent peers that obtained the torrent the new way to interoperate with old peers even if the torrent itself was created in a hybrid format.

But forever lugging around the legacy data on hybrid torrents created during the transition phase is not ideal either.

Hrm, I'll have to think about it.